reactos/dll/win32/browseui/shellbrowser.cpp

```diff
@@ -1254,13 +1343,13 @@ HRESULT CShellBrowser::DoFolderOptions()
     if (FAILED_UNEXPECTEDLY(hResult))
         return E_FAIL;
 
-    if (fCurrentShellView != NULL)
-    {
-        hResult = fCurrentShellView->AddPropertySheetPages(
-            0, AddFolderOptionsPage, reinterpret_cast<LPARAM>(&m_PropSheet));
-        if (FAILED_UNEXPECTEDLY(hResult))
-            return E_FAIL;
-    }
+    //if (fCurrentShellView != NULL)
+    //{
+    //    hResult = fCurrentShellView->AddPropertySheetPages(
+    //        0, AddFolderOptionsPage, reinterpret_cast<LPARAM>(&m_PropSheet));
+    //    if (FAILED_UNEXPECTEDLY(hResult))
+    //        return E_FAIL;
+    //}
```

Comment:
It was probably a test I forgot to revert and that came when I applied patches to git tree.